DDL-452 pulling device info on connection

RafalPolak opened the pull request:
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- checking other approach for fetch library
- add axios and check requests
- split handle prop into separate functions for get prop and dump sys for possibility to turn on/off on demand
- pass from add-device parameters for base url to know where send the request

RafalPolak committed:
DDL-452
- debugging connect

RafalPolak committed:
DDL-452
- WIP

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- leave header for application/json only

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- working on send request from webadb into stf api

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- checking other approach for fetch library

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- checking other approach for fetch library

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- checking other approach for fetch library

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- checking other approach for fetch library

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- checking other approach for fetch library

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- checking other approach for fetch library

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- checking other approach for fetch library

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- checking other approach for fetch library
- add axios and check requests

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- checking other approach for fetch library
- add axios and check requests

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- checking other approach for fetch library
- add axios and check requests

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- checking other approach for fetch library
- add axios and check requests

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- checking other approach for fetch library
- add axios and check requests
- split handle prop into separate functions for get prop and dump sys for possibility to turn on/off on demand
- pass from add-device parameters for base url to know where send the request

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- checking other approach for fetch library
- add axios and check requests
- split handle prop into separate functions for get prop and dump sys for possibility to turn on/off on demand
- pass from add-device parameters for base url to know where send the request

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- checking other approach for fetch library
- add axios and check requests
- split handle prop into separate functions for get prop and dump sys for possibility to turn on/off on demand
- pass from add-device parameters for base url to know where send the request

RafalPolak committed:
Update zerotier.tsx

comment dumpsys

RafalPolak committed:
Update zerotier.tsx

owocky commented on packages/demo/src/routes/zerotier.tsx:
```diff
@@ -43,17 +48,38 @@ export const ZeroTier = withDisplayName('ZeroTier')(({
         }
     }, [])
 
+    useEffect(() => {
+        if (isGetProp) {
+            setIsProp(false);
+            handleProp()
+                .then(() => {})
+                .catch(err => {
+                    console.error('Not able to handle properties: ', err)
+                })
+        }
+    })
```
Ten useEffect nie bedzie sie odpalal cały czas? nie powinien miec na koncu [] zeby sie wykonal tylko raz albo [isGetProp] zeby sie wykonywal tylko jak sie zmieni ten isGetProp ?

RafalPolak replied:
Tak i dlatego dawałem tego if'a tam. Dodałem tak jak sugerowałeś [isGetProp] aby wiedzieć czy już można robić zrzut czy nie. Samo [] raczej nie przejdzie.

RafalPolak committed:
DDL-452
- retrieve username from url
- set serial/username/date/prop/dumpsys for sending to API
- send dumpsys into device and then load into variable
- working on gzip data
- divide request get prop and dump sys
- check other solution for the fetch
- set date as a ISO string
- change parameter from user into email
- checking other approach for fetch library
- add axios and check requests
- comment dumpsys for now
- add isGetProp as dependency for effect
